lte/protos/session_manager.proto

```diff
@@ -1002,7 +1002,7 @@ message UPFPagingInfo {
 
 //Set Interface from Pipelined
 service SetInterfaceForUserPlane {
-    rpc SetUPFNodeState(UPFNodeState) returns (magma.orc8r.Void) {}
+    rpc SetUPFNodeState(UPFNodeState) returns (SmContextVoid) {}
```

Comment:
What is the difference?

Reply:
1) magma.orc8r.Void is simple generic empty message defined as below:
 message Void {
}

2) The file lte/protos/session_manager.proto has the information like below,
// TODO This supposed to be empty message and the element will be removed later
message SmContextVoid {
  string output = 1;
}
Also from AMF side N11 interface code expecting SmContextVoid, so to compatible with AMF added this SmContextVoid here